### ISSUE TITLE
docs: point to versioning ordering in packaging guide

### ISSUE THREAD

**Issue by c-cube** [ocaml/opam#3220]
https://www.debian.org/doc/debian-policy/#s-f-version should be easy to find when one picks a version number of their package.

**Comment by AltGr** [comment]
indeed. Thanks, now is a good time since that packaging guide needs rewriting!
That link or our [simplified version](http://opam.ocaml.org/doc/2.0/Manual.html#version-ordering), which doesn't mention "epochs" that we don't use and should hopefully be a little easier to figure out.
